features/add_cart.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Add dish to cart
+  As a visitor
+  In order to select a dish that I´d like to order
+  I would like to place it in the cart
+
+  Background:
+   Given the following restaurants exist
+    | name        |
+    | NisseKebbab |
+   And the following menus exist
+    | name            | restaurant  |
+    | Nisses Takeaway | NisseKebbab |
+   And the following dishes exist
+    | name       | price | menu            |
+    | Kebabrulle | 180   | Nisses Takeaway |
+    | Falafel    | 190   | Nisses Takeaway |
+
+  Scenario: I buy a dish and put it in the cart
+    Given I am on the restaurant menu page for "Nisses Takeaway"
+    Then I should see "Kebabrulle"
+    And I should see "Falafel"
+    And I click "Buy" on "Kebabrulle"
+    And I should see "Kebabrulle has been added to your order"
+    Then I should see "1 dishes in cart"
+
+  Scenario: I buy a second dish and put it in the cart
+    Given I am on the restaurant menu and have one item in my cart
```

Comment:
Instead of creating an entirely new step definition for this, I would prefer you just extend the first scenario. So you get down to `Then I should see "1 dishes in cart"` and then go ahead and add a second dish in the same scenario - combining these two. Then you can delete this step definition.